diff --git a/status.c b/status.c.new
index 18f0a1b..9d889a2 100644
--- a/status.c
+++ b/status.c.new
@@ -89,7 +89,7 @@ print_status(struct mpd_connection *conn)
 		       mpd_status_get_update_id(status));
 
 	if (mpd_status_get_volume(status) >= 0)
-		printf("volume:%3i%c   ", mpd_status_get_volume(status), '%');
+		printf("volume: %3i%c   ", mpd_status_get_volume(status), '%');
 	else {
 		printf("volume: n/a   ");
 	}
